hardware/flip-out-mechanisms.md

```diff
@@ -10,3 +10,18 @@ A lot of teams will make flip out mechanisms that are triggered by the robot's l
 
 Starstruck was a game that required a flip-out mech for the popular claw design.Many flip out mechanisms were triggered by the claws opening. Similar use of the intake to trigger a flip out is good for other games too.
 
+One example of several flip-out mechanisms being utilized on one robot was [97934V's Starstruck Deploy](https://www.youtube.com/watch?v=u9XEYpPA-H4). Most notably, its claw was held behind its lift tower until the lift was raised.
+
+![97934V's Starstruck Deploy](https://user-images.githubusercontent.com/65926085/88546554-2e072480-cfe2-11ea-8c75-ad71f8bb1d9d.png)
```

Comment:
For images on gitbook you must upload your image into the assets folder of this repo and then use the image like so: `
![5225a Lift](../../.gitbook/assets/5225alift.png)`